server/src/main/java/org/tctalent/server/model/db/FamilyRsdEvidenceForm.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2025 Talent Catalog.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+package org.tctalent.server.model.db;
+
+import org.tctalent.server.security.AuthService;
+import org.tctalent.server.service.db.CandidatePropertyService;
+import org.tctalent.server.service.db.CandidateService;
+
+/**
+ * Candidate portal form that captures refugee status documentation for
+ * relocating family members. The payload is stored as JSON in a candidate
+ * property so it can flexibly represent the per-member data set and any
+ * associated uploads.
+ */
+public class FamilyRsdEvidenceForm extends CandidateFormInstanceHelper {
+
+  private static final String FAMILY_RSD_JSON = "familyRsdEvidenceJson";
```

Comment:
It is up to you to be consistent with the property names. Here you name this property as "familyRsdEvidenceJson"

But in the Property definition below you name the property differently.

I think that the above line should be:
private static final String FAMILY_RSD_JSON = "FAMILY_RSD_EVIDENCE_INFO";